Automattic/jetpack
# [Subscription block] Add social followers toggle

#### Changes proposed in this Pull Request:
This PR adds a toggle to the Subscription block settings, that allows to toggle between including social followers (default) in the count, or excluding them.

<img width="283" alt="CleanShot 2022-11-16 at 16 02 36@2x" src="https://user-images.githubusercontent.com/528287/202216226-056d5e34-84f3-4236-9f92-0571e6c9d6b1.png">

Jetpack product discussion
pdDOJh-ZI-p2
pdDOJh-Yp-p2

Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1. Apply this PR
2. The checkbox should be visible & toggling it should update the number in both editor & frontend.

To test with simple sites:
- On your sandbox:
  - `bin/jetpack-downloader test jetpack add/subscription-block-social-toggle`(don't checkout trunk)

To test with Atomic:
- Follow these instructions: pb5gDS-1rQ-p2
 
Also make sure that users see the same count when upgrading (standard behavior should be to include social followers)
